src/pdb/mod.rs

```diff
@@ -368,9 +373,9 @@ pub struct RowGroup {
     rows: Vec<RowOffset>,
     row_presence_flags: u16,
     /// Unknown field, probably padding.
-    #[br(temp)]
-    #[bw(calc = 0)]
-    padding: u16,
+    ///
+    /// Apparently this is not always zero, so it might also be something different.
+    unknown: u16,
```

Comment:
can we still assert that its zero so we can more easily catch occurrences of this?

Reply:
No, that makes tests fail:

```
thread 'pdb_demo_tracks_PIONEER_rekordbox_export_pdb' panicked at 'called `Result::unwrap()` on an `Err` value:
 ╺━━━━━━━━━━━━━━━━━━━━┅ Backtrace ┅━━━━━━━━━━━━━━━━━━━━╸

 0: Error: unknown == 0 at 0x2fee
           While parsing field 'row_groups' in Page
     at src/pdb/mod.rs:282

 ╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸

', /home/jan/Projects/rekordcrate/target/debug/build/rekordcrate-2debded95e6c6d26/out/tests_pdb.rs:23:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Mhmm, I thought this might not be that common. Then I guess we leave the assert out and try to decipher the meaning of these fields some other time.

Reply:
Mhmm, I thought this might not be that common. Then I guess we leave the assert out and try to decipher the meaning of these fields some other time.